packages/vitest/src/integrations/chai/index.ts

```diff
@@ -53,6 +53,10 @@ export function createExpect(test?: Test) {
     return assert
   }
 
+  expect.unreachable = (message?: string) => {
+    chai.assert.fail(`expected ${message ? `"${message}"` : ''} not to be reached`)
```

Comment:
I think if you don't pass a name, it will have double space.

Reply:
😅You are right!